master/docs/developer/cls-buildsteps.rst

```diff
@@ -584,7 +584,7 @@ This class can only be used in new-style steps.
         Determine if the given path exists on the worker (in any form - file, directory, or otherwise).
         This uses the ``stat`` command.
 
-    .. py:method:: runGlob(glob)
```

Comment:
Do you prefer `runGlob(test)` to `runGlob(pattern)` or `runGlob(path)`? The other sections of help here use:

```
runXXX(param1, param2, ...)
   :param param1 some docs
   :param param2 some docs
```
so runGlob(path) might be more consistent. (Or runGlob(glob) if we end up changing the worker protocol).